solidity/test/bridge/VendingMachine.test.ts

```diff
@@ -1,25 +1,33 @@
-const { expect } = require("chai")
-const {
-  increaseTime,
+import { ethers, waffle, helpers } from "hardhat"
+import { expect } from "chai"
+import type { Signer } from "ethers"
+import { vendingMachineDeployment, constants } from "../fixtures"
+// eslint-disable-next-line import/extensions
```

Comment:
We do not need to do that in [random beacon tests](https://github.com/keep-network/keep-core/blob/874e247e8f21cec976f93b38604c37a1f77b8872/solidity/random-beacon/test/BLS.test.ts#L5). Did we miss some config rule?

Reply:
I suspect that in case of `random-beacon` the `lint` check is run after contracts are compiled and there are files generated in the ` typechain` folder. While here it is run before we generate `typechain`.

If you checkout a fresh `random-beacon` repo and call `yarn lint` there, you will get the same error.